tools/nightly-builder/build-required.js

```diff
@@ -0,0 +1,49 @@
+"use strict"
+
+const strftime     = require('strftime').timezone(0)
+    , after        = require('after')
+    , latestBuild  = require('./latest-build')
+    , latestCommit = require('./latest-commit')
+
+const dateFormat = '%Y%m%d'
+
+
+function timeString () {
+  return strftime(dateFormat)
+}
+
+
+function buildRequired (type, callback) {
+  let done = after(2, onData)
+    , build
+    , commit
+
+  function onData (err) {
+    if (err)
+      return callback(err)
+
+    let buildCommit = build.commit.substr(0, 10)
+      , buildDate   = strftime(dateFormat, build.date)
```

Comment:
unused, I guess the intention was originally to check both, but that seems unnecessary as per below.